### ISSUE TITLE
[Bug]Empty uBlock Settings section sometimes makes Fenix crash

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8202]
## Prerequisites
Have uBlock Origins installed

## Steps to reproduce
1. Enable/Disable uBlock repeatedly
2. While enable/disable, quickly tap the Settings section

### Expected behavior
The Settings section should be properly loaded.

### Actual behavior
Blank Settings section.
Sometimes **Fenix crashes**.

### Device information

* Android device: 

• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7.0.0)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

* Fenix version: Nightly #20380605 from 2/7

### Notes
► Video
![20200208_201235](https://user-images.githubusercontent.com/51314259/74031728-ee5c6a00-49ba-11ea-936f-f9848572f706.gif)

► Log
```kotlin
    --------- beginning of crash
02-07 14:29:44.782 29080-29080/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.nightly, PID: 29080
    java.lang.IllegalStateException: ScrollView can host only one direct child
        at android.widget.ScrollView.addView(ScrollView.java:252)
        at com.google.android.material.snackbar.BaseTransientBottomBar$1.handleMessage(BaseTransientBottomBar.java:38)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:152)
        at android.app.ActivityThread.main(ActivityThread.java:5507)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
02-07 14:29:44.782 29080-29080/? D/AppTracker: App Event: crash
02-07 14:29:44.792 1328-1365/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@856b13c
02-07 14:29:44.792 1328-1365/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@751eec5
02-07 14:29:44.801 1328-1365/? W/ActivityManager:  Dismiss app error dialog : org.mozilla.fenix.nightly
02-07 14:29:44.802 1328-1365/? W/AppOps: Finishing op nesting under-run: uid 1000 pkg android code 24 time=0 duration=0 nesting=0
02-07 14:29:44.807 29080-29080/? I/Process: Sending signal. PID: 29080 SIG: 9
02-07 14:29:44.821 1328-29516/? D/DropBoxManagerService: file :: /data/system/dropbox/data_app_crash@2020-02-07-14_29_44_793.txt
02-07 14:29:44.884 1328-2578/? I/WindowState: WIN DEATH: Window{83ce3e6 u0 SurfaceView}
02-07 14:29:44.884 1328-2138/? D/GraphicsStats: Buffer count: 7
02-07 14:29:44.885 1328-1349/? I/ActivityManager: Process org.mozilla.fenix.nightly (pid 29080) has died
02-07 14:29:44.886 1328-1718/? W/InputDispatcher: channel '61fb97b org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
02-07 14:29:44.886 1328-1718/? E/InputDispatcher: channel '61fb97b org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App (server)' ~ Channel is unrecoverably broken and will be disposed!
02-07 14:29:44.886 29164-29164/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-07 14:29:44.889 1328-1350/? I/WindowState: WIN DEATH: Window{61fb97b u0 org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App}
02-07 14:29:44.889 29148-29148/? I/ServiceChildProcess: Service has been unbound. Stopping.
02-07 14:29:44.889 1328-1350/? W/InputDispatcher: Attempted to unregister already unregistered input channel '61fb97b org.mozilla.fenix.nightly/org.mozilla.fenix.nightly.App (server)'
02-07 14:29:44.909 1328-1349/? V/BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@8bc317d
02-07 14:29:44.909 1328-1349/? I/ActivityManager: Start proc 29517:org.mozilla.fenix.nightly:mozilla.components.lib.crash.CrashReporter/u0a212 for activity org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity
02-07 14:29:44.912 29148-29148/? I/Process: Sending signal. PID: 29148 SIG: 9
02-07 14:29:44.913 1328-2222/? I/ActivityManager: Process org.mozilla.fenix.nightly:socket (pid 29164) has died
02-07 14:29:44.971 1328-1363/? V/ActivityManager: killProcessGroupAsync took 37 ms for PID 29080 on thread 14
02-07 14:29:44.972 1328-1363/? V/ActivityManager: killProcessGroupAsync took 0 ms for PID 29164 on thread 14
02-07 14:29:44.974 1328-2578/? I/ActivityManager: Process org.mozilla.fenix.nightly:tab (pid 29148) has died
02-07 14:29:44.975 1328-1363/? V/ActivityManager: killProcessGroupAsync took 0 ms for PID 29148 on thread 14
02-07 14:29:45.038 1328-2605/? W/InputMethodManagerService: Got RemoteException sending setActive(false) notification to pid 29080 uid 10212
02-07 14:29:45.231 29517-29517/? D/AppTracker: App Event: start
02-07 14:29:45.246 29517-29533/? D/OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
02-07 14:29:45.290 29517-29533/? I/Adreno: QUALCOMM build                   : 19f8f74, Iac38d51750
    Build Date                       : 06/06/16
    OpenGL ES Shader Compiler Version: XE031.07.00.01
    Local Branch                     : case2489916_Unmap_crash
    Remote Branch                    : 
    Remote Branch                    : 
    Reconstruct Branch               : 
02-07 14:29:45.300 29517-29533/? I/OpenGLRenderer: Initialized EGL, version 1.4
02-07 14:29:45.328 29517-29517/? D/AbstractTracker: Event success
02-07 14:29:45.342 29517-29533/? E/HAL: load: gralloc.msm8996.so
02-07 14:29:45.379 1328-1387/? I/ActivityManager: Displayed org.mozilla.fenix.nightly/mozilla.components.lib.crash.prompt.CrashReporterActivity: +487ms
```

**Comment by csadilek** [comment]
We should disable the settings menu item for disabled extensions.

**Comment by csadilek** [comment]
@AndiAJ OK this should be fixed now by making sure we don't show the settings menu item when it's not available (same behaviour as on desktop) and even if you're fast enough clicking on it, the crash should not be reproducible anymore. :)

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly #20410606 from 2/10`** using the following devices:
• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Sony Xperia ZX (Android 7.0.0)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video
![20200210-173508](https://user-images.githubusercontent.com/51314259/74164157-0d156780-4c2c-11ea-8c3a-e4237227a0f1.gif)
